examples/viewer.html

```diff
@@ -5,6 +5,7 @@
   <title>Viewer</title>
   <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
   <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async src="/dist/v0/examples/amp-viewer-host.max.js"></script>
```

Comment:
This, unfortunately, should be sync (remove `async`). This is because you use this file directly in the sync code below. There's a way to make it all async, of course. But I think it'd be overkill.